Code/GraphMol/SmilesParse/catch_tests.cpp

```diff
@@ -2204,3 +2214,71 @@ M  END
     }
   }
 }
+
+TEST_CASE("ring bond stereochemistry in CXSMILES") {
+  SECTION("basic reading") {
+    std::vector<std::pair<std::string, Bond::BondStereo>> tests = {
+        {"C1CCCC/C=C/CCC1 |t:5|", Bond::BondStereo::STEREOTRANS},
```

Comment:
The test code for CXSMILES/CXSMARTS for reactions is in the `ChemReactions` directory since I didn't want to add a circular dependency:
https://github.com/rdkit/rdkit/blob/master/Code/GraphMol/ChemReactions/catch_tests.cpp#L1058

I will add a test there.

Reply:
oh, good thing you noticed that. Neither this code nor the code for wedged bonds works properly in reactions.
I'm fixing it and adding tests.